Libraries/Network/XMLHttpRequest.js

```diff
@@ -359,39 +398,37 @@ class XMLHttpRequest extends EventTarget(...XHR_EVENTS) {
     ));
     this._subscriptions.push(RCTNetworking.addListener(
       'didReceiveNetworkData',
-      (args) =>  this.__didReceiveData(...args)
+      (args) => this.__didReceiveData(...args)
+    ));
+    this._subscriptions.push(RCTNetworking.addListener(
```

Comment:
I'm not super familiar with all of this code: does this mean we always send all the incremental data chunks **and** the complete response when the request finishes? That seems wasteful.